apollo-client/src/main/java/com/ctrip/framework/apollo/ConfigFile.java

```diff
@@ -35,6 +35,14 @@ public interface ConfigFile {
    */
   boolean hasContent();
 
+  /**
+   * Get the appId of this config file instance
+   * @return the appId
+   */
+  default String getAppId(){
+    return null;
+  }
+
```

Comment:
```suggestion
  /**
   * Get the appId of this config file instance
   * @return the appId
   *
   * @since 2.4.0
   */
  default String getAppId(){
    return null;
  }

```